go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -367,3 +374,39 @@ func verifyDisableEnableRedoLogs(ctx context.Context, t *testing.T, mysqlSocket
 		}
 	}
 }
+
+// This helper function wait for all replicas to catch-up the replication.
+// It does this by querying the status detail url of each replica and find the lag.
+func waitForReplicationToCatchup(tablets []cluster.Vttablet) bool {
+	endTime := time.Now().Add(time.Second * 30)
+	timeout := time.After(time.Until(endTime))
```

Comment:
Nit, but better to use a Timer or a Ticker with a context.WithTimeout. This is fine though as it's only test code.